spec/spec_helper.rb

```diff
@@ -116,14 +116,7 @@
   config.add_setting :reload_page_wait_time, default: 60
 end
 
-chromedriver_from_path = File.which("chromedriver")
-
-if chromedriver_from_path
-  # Use the installed chromedriver, rather than chromedriver-helper
-  Selenium::WebDriver::Chrome.driver_path = chromedriver_from_path
-else
-  require "chromedriver-helper"
-end
```

Comment:
Even if you switch to using Webdrivers, it would be good to keep this in, as it makes it possible to run the Publishing E2E tests in environments where Webdrivers doesn't work (I'm not sure the binaries it downloads would work within the govuk-guix setup for example).

Reply:
I've decided to delay the upgrade to `webdrivers` from `chromedriver-helper` because it opened up a whole new can of worms. We'll save that for another day and hopefully in the meantime, at least have some working e2e tests.